tensorflow/compiler/xla/tests/convolution_variants_test.cc

```diff
@@ -34,6 +34,7 @@ limitations under the License.
 #include "tensorflow/compiler/xla/tests/literal_test_util.h"
 #include "tensorflow/compiler/xla/tests/test_macros.h"
 #include "tensorflow/compiler/xla/xla_data.pb.h"
+#include "tensorflow/tsl/platform/tensor_float_32_utils.h"
```

Comment:
This header include needs a corresponding BUILD file change.

Reply:
I've updated the BUILD file with the following, which seems to work to build both CPU and GPU tests.
```
select({
        "//tensorflow/tsl:is_cuda_enabled": [
            "//tensorflow/tsl/platform:tensor_float_32_hdr_lib",
        ],
        "//conditions:default": [
            "//tensorflow/tsl/platform:tensor_float_32_utils",
        ],
    }),
```